UITextFieldPickerInput/PickerInputSettings.swift

```diff
@@ -20,7 +20,9 @@ public struct PickerInputSettings {
     public var hideCancelButton: Bool = false
     public var isToolBarHidden: Bool = false
     public var hideWhenSelect: Bool = false
-    
+    public var hideWhenTouchOverlay: Bool = false
+    public var overlayViewBackgroundColor: UIColor = UIColor.black.withAlphaComponent(0.6)
+
     public init(cancelButtonTitle: String = "Cancel",
```

Comment:
Instead of sending a lot of property variables, I would suggest to create one object, that represents those Options you need and to pass that, what do you think?

Reply:
The PickerInputSettings is already that type of object 😅